definitions/npm/ramda_v0.x.x/flow_v0.49.x-/ramda_v0.x.x.js

```diff
@@ -430,8 +430,8 @@ declare module ramda {
   declare function type(x: ?any): string;
   declare function isArrayLike(x: any): boolean;
 
-  declare function isNil(x: void | null): true;
-  declare function isNil(x: mixed): false;
+  declare function isNil(x: mixed): boolean %checks(x === undefined ||
```

Comment:
Is `%` meant to be a `//`?

Reply:
Nope. It's a special notation used within flow to [add refinement of types](https://github.com/facebook/flow/issues/4723#issuecomment-325157852). Since we don't have the actual function definition here, we have to provide some additional hints as to what exactly it's checking. Fortunately in the case of `isNil`, that's quite simple.

Reply:
Well TIL!

Reply:
Neat, right? The docs for more advanced features of flow are disappointingly sparse. 😦

Reply:
Super neat! It's a shame about the docs, but with flow changing so often it's hard to blame the team. Heck I have a hard time documenting at work, and I'm paid to do it. Hopefully once everything settles and they release a V1, we'll get some solid docs 📖